src/dashboard/components/files/FileActions.js

```diff
@@ -29,15 +37,46 @@ export default function FileOptions ({missing, id, filePath, openFile}) {
     return <DeleteConfirmModal name={name} onDelete={deleteFile} onCancel={() => setDeleting(false)}/>
   }
 
+  const renameFile = () => {
+    console.log('start function')
+    // first, i need to open the dialog to ask the user where to save the file
+    const fileName = dialog.showSaveDialogSync(win, {filters, title: i18n('Where would you like to move it?')})
```

Comment:
so this `i18n` can actually be just `t` (for translate).

I'm trying to move all the uses of i18n to t